src/scrollspy.ts

```diff
@@ -190,24 +200,27 @@ export class ScrollSpy extends Component<ScrollSpyOptions> {
     // remember elements in view for next tick
     ScrollSpy._elementsInView = intersections;
     if (ScrollSpy._elements.length) {
-      const options = (ScrollSpy._elements[0].el as any).M_ScrollSpy.options;
+      const options = ScrollSpy._elements[0].el['M_ScrollSpy'].options;
```

Comment:
I think we might need to revert this, since it introduces backwards compatibility issues with applications using the previous object orientated approach, propose to just set it to warnings until we can declare the correct type definition